molecule/_shared/base.yml

```diff
@@ -2,16 +2,12 @@
 scenario:
   test_sequence:
     - lint
-    # - destroy
-    # - dependency
     - syntax
     - create
-    # - prepare
+    - prepare
```

Comment:
added prepare step where molecule install required packages (unzip and netaddr)
they should be installed before we run ansible-playbook because it read modules only once while running